spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewritePositionDeleteFilesProcedure.java

```diff
@@ -41,19 +41,30 @@ public TestRewritePositionDeleteFilesProcedure(
   }
 
   private void createTable() throws Exception {
+    createTable(false);
+  }
+
+  private void createTable(boolean partitioned) throws Exception {
+    String partitionStmt = partitioned ? "PARTITIONED BY (id)" : "";
```

Comment:
maybe pass in partitionStatement instead of boolean? It seems more straightforward imo

Reply:
Hm, I think it would be shorter to keep it boolean?  There's a lot of test that have util methods that take in ```boolean partitioned```

Is it not straightforward because it implicitly chooses the partition column for the caller?  I think to me, its fine as anyway the schema is hidden from user.